src/sentry/api/urls.py

```diff
@@ -971,7 +971,7 @@
                 url(
                     r"^(?P<organization_slug>[^\/]+)/projects-count/$",
                     OrganizationProjectsCountEndpoint.as_view(),
-                    name="sentry-api-0-organization-projects",
+                    name="sentry-api-0-organization-projects-count",
```

Comment:
I changed this because the name was exactly the same as `OrganizationProjectsEndpoint` and it was conflicting